platform/vcs-impl/src/com/intellij/openapi/vcs/actions/AnnotateToggleAction.java

```diff
@@ -56,6 +56,8 @@
 public class AnnotateToggleAction extends ToggleAction implements DumbAware {
   public static final ExtensionPointName<Provider> EP_NAME =
     ExtensionPointName.create("com.intellij.openapi.vcs.actions.AnnotateToggleAction.Provider");
+  public static final ExtensionPointName<ExtendedActiveAnnotationGutter> EP_NAME_0 =
```

Comment:
Can be moved to the `ExtendedActiveAnnotationGutter` itself (this will also allow to avoid ambigous naming).